cmd/main.go

```diff
@@ -112,16 +113,17 @@ func parseConfig() (spirev1alpha1.ControllerManagerConfig, ctrl.Options, []*rege
 
 	options := ctrl.Options{Scheme: scheme}
 	var ignoreNamespacesRegex []*regexp.Regexp
+	var parentIDTemplate *template.Template = nil
```

Comment:
nit: assignment to `nil` is unnecessary; variables that are not initialized default to their zero value (i.e. nil for pointer types)